lib/hanami/helpers/form_helper/form_builder.rb

```diff
@@ -1676,7 +1676,7 @@ def _select_input_name(name, multiple)
         # @api private
         def _select_option_selected?(value, selected, input_value, multiple)
           if input_value && selected.nil?
-            value.to_s == input_value
+            value.to_s == input_value.to_s
```

Comment:
WDYT about `String(input_value)`?

Reply:
`String(s)` seems to only call `#to_s`, so I don't think this would be beneficial.